src/app/gagestats/gagestats.component.html

```diff
@@ -13,14 +13,14 @@ <h4>{{timestamp | date:"MMMM d, y hh:mma"}}</h4>
     <span MathJax class="hidden" id="mathJaxLoad">`Preloading`</span> 
     <div class="col-md-6" style="padding: 0;">
         <div *ngIf="showStationType" class="app-body-section">
-            <label class="big">Station Type:</label>
-            <h5 class="bold">{{selectedStationType.name}}</h5>  
+            <label class="big">Station Types:</label>
+            <h5 class="bold">{{selectedStationType}}</h5>  
```

Comment:
It'd be nice to show the name here instead, but that could be a separate issue

Reply:
For some reason I have yet to figure out, selectedStationType is a number rather than an array. Because of this `selectedStationType.name` won't return anything. I removed this pop-up from the main gage stats page. Also, in a later pull request Harper added a function which calls the station type for each selected station and displays this in the station table.